app/controllers/admin/questions_controller.rb

```diff
@@ -53,13 +53,13 @@ def update
   def destroy
     Question.find(params[:id]).destroy
     flash[:success] = 'Question deleted'
-    redirect_to admin_questions_path
+    redirect_to :back
```

Comment:
it's not mistake - but better use explicit redirect (like was before admin_questions_path)